web/src/components/navbar/NavBarMobile.tsx

```diff
@@ -15,9 +15,16 @@ interface Props {
   task?: Task;
   hideMiniRoadmap?: boolean;
   showSidebar?: boolean;
+  landing?: boolean;
 }
 
-export const NavBarMobile = ({ scrolled, task, showSidebar, hideMiniRoadmap }: Props): ReactElement => {
+export const NavBarMobile = ({
+  scrolled,
+  task,
+  showSidebar,
+  hideMiniRoadmap,
+  landing,
```

Comment:
I lean towards using deconstructed props, but I think we as a team are trying to use a props object to make it more obvious where a value is coming from. Not a necessary change, but this might be a good chance to change that pattern in this component to be consistent with the other components in our app.

Reply:
+1

My argument against deconstructed props is that seeing `props.scrolled` in the body of JSX makes it 100% obvious where that value comes from without "clicking through" to its definition.

Whereas if you deconstruct and you see `scrolled` variable being used, you have no idea if that is computed or passed it. You have to click through to see if it comes from props, or if it is defined somewhere in the component `const scrolled = ....`

It is useful to know at-a-glance whether something is computed or passed from props, thus my argument against deconstructing.

Agree with JP, this is a good time to apply the campground rule